src/content/self-hosted/administration/configuration.mdx

```diff
@@ -261,6 +265,8 @@ cloud:
 
 ##### IAM Role for Service Account (IRSA)
 
+Create a IAM role with access to the Okteto Registry bucket, bind the IAM role to the Okteto Registry Kubernetes service account and apply these changes to the Okteto configuration:
```

Comment:
Can we also tell the reader the exact name of the service account? Or show them how to get it?